nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -615,92 +615,94 @@ in
 
   };
 
-  config = {
-    assertions = [
+  # store them in lib so we can mkImageMediaOverride the
+  # entire file system layout in installation media (only)
```

Comment:
Could you explain why this isn't a module? Maybe it could still be a function returning a module if that's clearer. Not sure. Maybe someone else has a preference?

Reply:
I was just looking for a way of how to _borrow_ otherwise _owned_ data to a downstream module. I thine `config.lib` is best suited for this vertical bush radio.